src/types/spicetify.d.ts

```diff
@@ -1997,10 +1997,12 @@ declare namespace Spicetify {
 	 */
 	namespace Config {
 		const version: string;
-		const current_theme: string;
-		const color_scheme: string;
+		let current_theme: string;
+		let color_scheme: string;
 		const extensions: string[];
 		const custom_apps: string[];
+		let local_theme : string;
+		let local_color_scheme : string;
```

Comment:
why do you directly modify the spicetify types, don't do it please

Reply:
Or it is better to modify the types in the spicetify cli repo

Reply:
Well, technically the `Config` object shouldn't be used for marketplace usage. It should be in its own object that only marketplace uses

Reply:
I think it was done this way to keep compatibility with extensions that check current theme and themes that check current color scheme

Reply:
but they don't check the `local_theme` nor `local_color_scheme`, so this stuff could be kept in local storage for example

Reply:
Yes, `local_theme` and `local_color_scheme` definitely shouldn't be a part of the spicetify types declaration file